example/ios/CameraRollExample/Info.plist

```diff
@@ -26,8 +24,23 @@
 	<string>1</string>
 	<key>LSRequiresIPhoneOS</key>
 	<true/>
+	<key>NSAppTransportSecurity</key>
+	<dict>
+		<key>NSAllowsArbitraryLoads</key>
+		<true/>
+		<key>NSExceptionDomains</key>
+		<dict>
+			<key>localhost</key>
+			<dict>
+				<key>NSExceptionAllowsInsecureHTTPLoads</key>
+				<true/>
+			</dict>
```

Comment:
> You must supply a justification during App Store review if you set the key’s value to YES, as described in [Provide Justification for Exceptions](https://developer.apple.com/documentation/security/preventing_insecure_network_connections#3138036). Use this key with caution because it significantly reduces the security of your app. 

([source](https://developer.apple.com/documentation/bundleresources/information_property_list/nsapptransportsecurity/nsallowsarbitraryloads))

Looks like the apple "review lords" may have issues with this key for security reasons, even if it's only applied on `localhost`. I saw we have it on `status-mobile` as well so should be fine, but wondering if using `NSAllowsLocalNetworking` instead would have the same effect and please the overlords.

Reply:
What is the need for this addition?

Reply:
Yeah, was meant for the app, since we have it there as well.